doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -341,15 +341,9 @@ In order to properly set up the data replication, the secondary nodes must be ab
 
    For more information on the management REST API, see the  Security Server User Guide \[[UG-SS](#13-references)\].
 
-10. Note about API keys and caching.
-   If API keys have been created for primary node, those keys are replicated to secondaries, like everything else from `serverconf` database is.
-   The keys that are associated with the `xroad-securityserver-observer` role have read-only access to the secondary.
-   Instead, the keys that are not associated with the `xroad-securityserver-observer` role, don't have any access to the secondary and API calls will fail.
-   To avoid this, secondary REST API should only be accessed using keys associated with the `xroad-securityserver-observer` role, and only for operations that read configuration, not updates. <p>
-   Furthermore, API keys are accessed through a cache that assumes that all updates to keys (e.g. revoking keys, or changing permissions) are done using the same node.
-   If API keys are changed on primary, the changes are not reflected on the secondary caches until the next time `xroad-proxy-ui-api` process is restarted.
-   To address this issue, you should restart secondary nodes' `xroad-proxy-ui-api` processes after API keys are modified (and database has been replicated to secondaries), to ensure correct operation.<p>
-   Improvements to API key handling in clustered setups will be included in later releases.
+10. <p> Note about API keys and caching. If API keys have been created for primary node, those keys are replicated to secondaries, like everything else from `serverconf` database is. The keys that are associated with the `xroad-securityserver-observer` role have read-only access to the secondary. Instead, the keys that are not associated with the `xroad-securityserver-observer` role, don't have any access to the secondary and APIcalls will fail. To avoid this, secondary REST API should only be accessed using keys associated with the `xroad-securityserver-observer` role, and only for operations that read configuration, not updates. </p>
+   <p> Furthermore, API keys are accessed through a cache that assumes that all updates to keys (e.g. revoking keys, or changing permissions) are done using the same node. If API keys are changed on primary, the changes are not reflected on the secondary caches until the next time `xroad-proxy-ui-api` process is restarted. To address this issue, you should restart secondary nodes' `xroad-proxy-ui-api` processes after API keys are modified (and database has been replicated to secondaries), to ensure correct operation.</p> 
+   <p>Improvements to API key handling in clustered setups will be included in later releases.</p>
```

Comment:
Here I think we don't want to use paragraphs as it seems to break the ordered list styling:

https://github.com/nordic-institute/X-Road/blob/4fa1e28c8bdf5a1bb51ecb8bbebba1b44aa781a2/doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md#33-secondary-installation

vs

https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md#33-secondary-installation